specification/vmware/Microsoft.AVS.Management/placementPolicies.tsp

```diff
@@ -28,33 +27,12 @@ interface PlacementPolicies {
     LroHeaders = Azure.Core.Foundations.RetryAfterHeader
   >;
 
-  // 202 response has a body, but it should not to match ArmCustomPatchAsync.
-  // update is ArmCustomPatchAsync<PlacementPolicy, PlacementPolicyUpdate>;
   #suppress "@azure-tools/typespec-azure-resource-manager/no-response-body"
-  #suppress "@azure-tools/typespec-azure-core/no-openapi" "DO NOT COPY - TODO migrate to LRO apis"
-  @doc("Update a PlacementPolicy")
-  @extension("x-ms-long-running-operation", true)
-  @extension(
-    "x-ms-long-running-operation-options",
-    #{ `final-state-via`: "location" }
-  )
-  @armResourceUpdate(PlacementPolicy)
-  @patch
-  update(
-    ...ResourceInstanceParameters<PlacementPolicy>,
-
-    @bodyRoot
-    @doc("The placement policy properties to be updated.")
-    placementPolicyUpdate: PlacementPolicyUpdate,
-  ): ArmResponse<PlacementPolicy> | {
-    ...AcceptedResponse;
-    ...Azure.Core.Foundations.RetryAfterHeader;
-    ...LocationHeader;
-
-    @bodyRoot
-    @doc("The updatd placement policy.")
-    placementPolicy: PlacementPolicy;
-  } | ErrorResponse;
+  update is ArmCustomPatchAsync<
+    PlacementPolicy,
+    PlacementPolicyUpdate,
+    Response = ArmResponse<PlacementPolicy> | ArmAcceptedLroResponseWithResponseBody<PlacementPolicy> | ErrorResponse
```

Comment:
The `| ErrorResponse` may be redundent (if you are using the one from Azure.ResourceManager lib).

The template should already have it as default
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/lib/operations.tsp#L315-L330

(but if change of this affect Swagger, then I would be wrong...)